src/main/java/com/iexec/common/config/PublicChainConfig.java

```diff
@@ -31,5 +31,9 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class PublicChainConfig {
+    private Integer chainId;
+    private boolean isSidechain;
+    private String nodeAddress;
+    private String hubAddress;
```

Comment:
I find it clearer this way
```suggestion
    private String chainNodeUrl;
    private String hubContractAddress;
```

Reply:
I agree, how about `iExecHubContractAddress`?

Reply:
`iExecHubContractAddress` would be better as we specifically target our contract. Thanks for the suggestions!